### PR TITLE
Complete admin user show page

### PR DESCRIPTION
**Pull request by grwthomps**

### CODE REVIEW

File: app/views/admin/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,4 @@
-<h1><%= @user.name %></h1>
+<%= render partial: 'partials/user_profile', locals:
+  {
+    admin: true
+  } %>
```
Comment (johnktravers) [inline app/views/admin/users/show.html.erb:4]: Excellent use of partials!

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,38 +1,4 @@
...
-<article class="justify-content-center row mt-4">
-  <% if !@user.orders.empty? %>
-    <%= link_to 'Your Orders', '/profile/orders', class: 'btn btn-success ml-2 mr-2' %>
-  <% end %>
-  <%= link_to 'Edit Your Info', '/profile/edit', class: 'btn btn-primary ml-2 mr-2' %>
-  <%= link_to 'Change Your Password', '/profile/edit?info=false', class: 'btn btn-danger ml-2 mr-2' %>
-</article>
+<%= render partial: 'partials/user_profile', locals:
+  {
+    admin: false
```
Comment (zacisaacson) [inline app/views/users/show.html.erb:3]: Liking these partials

File: app/views/admin/users/show.html.erb [hunk 3]
```diff
@@ -1 +1,4 @@
-<h1><%= @user.name %></h1>
+<%= render partial: 'partials/user_profile', locals:
+  {
+    admin: true
```
Comment (zacisaacson) [inline app/views/admin/users/show.html.erb:3]: Partials, yes.

File: app/views/partials/_user_profile.html.erb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  </tr>
+  <tr>
+    <td>Zip</td>
+    <td><%= @user.zip %></td>
+  </tr>
+</table>
+
+<article class="justify-content-center row mt-4">
+  <% if !@user.orders.empty? %>
+    <%= link_to 'Your Orders', '/profile/orders', class: 'btn btn-success ml-2 mr-2' %>
```
Comment (johnktravers) [inline app/views/partials/_user_profile.html.erb:34]: We'll have to change the route to the orders page for the admin

**Review by zacisaacson — APPROVED** [review]

**Review by johnktravers — COMMENTED** [review]
Awesome work!

**Pull request merged**